app/views/incident_reports/_show.haml

```diff
@@ -29,9 +29,6 @@
 %p
   %strong= human_name :incident, :bus
   = report.bus
-%p
-  %strong= human_name :incident_report, :occurred_at
-  = report.occurred_at_readable
```

Comment:
Would people still expect to see this here? Do we want to still display occurred at for the incident the report is associated with or..? This is a business logic question, I really don't know.